README.md

```diff
@@ -1,69 +1,40 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Hello Microverse
 
-> Hello Microverse! project.
+> In this project, you will set up a "Hello world" repository. No complex coding is required for this exercise. My goal here is to master all of the tools and best practices i have learned about in previous steps. I will be using them in all Microverse projects and most likely in my future job as well.
```

Comment:
- [ ] You should update the description section to reflect the new project. I believe this description is for the `Hello Microverse` project.